format/repositories.bzl

```diff
@@ -47,7 +47,7 @@ def rules_format_dependencies():
         sha256 = "978eaffdc3716bbc0859aecee0d83875cf3ab8d8725779448f0035309d9ad9f3",
         url = "https://github.com/nicklockwood/SwiftFormat/releases/download/0.49.17/swiftformat.zip",
         patch_cmds = [
-            "xattr -c swiftformat",
+            "if command -v xattr > /dev/null; then xattr -c swiftformat; fi",
```

Comment:
Curious. What is this doing? I am having trouble finding doc on `xattr -c`.

Reply:
on MacOS it clears the "application comes from an untrusted developer" bit

Reply:
oops I should have answered in the form of a comment in the code, will add